test/posix-mq1.c

```diff
@@ -62,13 +62,19 @@ parent(const char *mqname)
   fflush(stdout);
 
   int i = 1;
+  static int unlinked = 0;
   while (1) {
     printf("Server: %d\n", i);
     fflush(stdout);
     msg_snd(mqdes, i);
     sleep(1);
+    if (!unlinked) {
+      mq_unlink(mqname); /* parent and child will continue to use mqname */
```

Comment:
Putting `mq_unlink` changes the semantics of this test. In the previous version, we would ckpt/restore "unlinked" message-queues, whereas in this one, the first checkpoint will be without unlinking. I guess, we can add another test later.